orttraining/orttraining/models/runner/training_runner.cc

```diff
@@ -726,10 +732,13 @@ Status TrainingRunner::TrainingLoop(IDataLoader& training_data_loader, IDataLoad
   if (params_.perf_output_dir.empty()) {
     printf("No perf output directory specified, skipping save of trained perf metrics.\n");
   } else {
+    short average_CPU_usage = p_ICPUUsage->GetUsage();
```

Comment:
nit: these variables could be const

Reply:
nit: naming - how about "average_CPU_usage" -> "average_cpu_usage" and "p_ICPUUsage" -> "cpu_usage_calculator"?